### PR TITLE
Add style.css and index.html

### PR DESCRIPTION
**Pull request by AACHOURMOHAMED**
Add style.css and index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    </article> 
+    <article id="services">
+        <div class="icon">
+            <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+          </div>
+        <div class="content">
+            <h1 class="title">Premium Materials</h1>
+            <P class="text">For every purchase you make, we will ensure there are no damages or
+                faults and we will check and test the pitch of your instrument.
+            </P>
```
Comment (AACHOURMOHAMED) [inline index.html:76]: kindly use tag h1 only one time for best practices HTML 
check this link  [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

**Review by AACHOURMOHAMED — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
[cat-driving-serious](url)

# Changes required ♻

- [ ] kindly Add a description to the Readme file please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        </div>
+    </article>  
+    </div>
+    
+
+<!--the end of our services-->
+
+
```
Comment (ishimwezachee) [inline index.html:84]: - [OPTIOANAL ] You should review the spaces through your codes. Kindly review these space, and explain to students why its important to not add spaces in codes

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @AACHOURMOHAMED   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 

## Required Changes ♻️

- [ ] Could you review your Reame.md documentation. It is not professional

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

File: index.html (hunk 2 above)
Comment (AACHOURMOHAMED) [inline index.html:84]: [OPTIONAL ] kindly remove the space between your code to keep your work clean (clean code) Thanks

**Review by AACHOURMOHAMED — COMMENTED** [review]
# require changes  :recycle:

## Hi [@AACHOURMOHAMED ](https://github.com/ )👋


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

2DrZ



## Highlights 🌟

- ✔️ Descriptive commmit messsages
- :x:  Descriptive PR description
-  ✔️ No linter Errors

# require changes  :recycle:

- [ ] kindly Add a description to the Readme file please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.



Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@AACHOURMOHAMED](https://github.com/AACHOURMOHAMED)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @AACHOURMOHAMED , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AACHOURMOHAMED** [comment]
Thank you for approving the code review

**Pull request merged**